export const handleError = async () => {
  // some code
};
